src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/domain/documents/Et3ResponseData.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.domain.documents;
+
+import lombok.Data;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@Data
+@SuperBuilder
+@SuppressWarnings({"PMD.LinguisticNaming", "PMD.TooManyFields"})
+public class Et3ResponseData {
```

Comment:
Although it's a simple POJO, it should still have JavaDoc!
Explain a bit about what it's used for